Formula/mariadb.rb

```diff
@@ -13,6 +13,10 @@ class Mariadb < Formula
   devel do
     url "http://ftp.osuosl.org/pub/mariadb/mariadb-10.2.1/source/mariadb-10.2.1.tar.gz"
     sha256 "90b7a17f3372c92c12dff084b37fcca8c4cf8106f4dcabd35fadc8efbaa348a2"
+
+    # upstream fix for compilation error
+    # https://jira.mariadb.org/browse/MDEV-10322
+    patch :DATA
```

Comment:
Unfortunately not yet. 10.2.2 is, according to their JIRA, targeted for September 1st.